src/components/sections/planner/plannerandinfos/PlannerTabs.jsx

```diff
@@ -59,7 +59,7 @@ class PlannerTabs extends Component {
     const { user, setPlannersDispatch } = this.props;
 
     if (!user) {
-      setPlannersDispatch([]);
+      // setPlannersDispatch([]);
```

Comment:
(그냥 코멘트) 이 부분은 일단 주석 처리해두고 넘어가고, 나중에(..) planner 쪽 컴포넌트 리팩토링할 때 재확인 후 지우면 될 것 같습니다 👍

Reply:
확인해보니 user 정보 없을 때 최초로 `[]` 세팅해줘야 해서 필요한 거였습니다 ㅇ0ㅇ 주석 처리하면 안 되네요!

```suggestion
      setPlannersDispatch([]);
```

Reply:
reducers/planner/planner.ts 에서 PlannerState의 initialState를 planners: null 에서 planners: [] 로 변경해서 setPlannersDispatch([]);  코드가 필요없어졌는데,
원복하기로 결정하였습니다.